src/main/java/io/wisetime/idea/branch/settings/SettingsPanel.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2021 Practice Insight Pty Ltd. All Rights Reserved.
+ */
+
+package io.wisetime.idea.branch.settings;
+
+import com.intellij.openapi.ui.VerticalFlowLayout;
+import com.intellij.ui.components.JBCheckBox;
+import javax.swing.JPanel;
+
+/**
+ * @author vadym
+ */
+public class SettingsPanel {
+
+  private final JBCheckBox windowTitle;
+  private final JBCheckBox projectName;
+  private JPanel root;
+
+  public SettingsPanel() {
+    root = new JPanel(new VerticalFlowLayout());
+    windowTitle = new JBCheckBox("Append VCS branch name to window title");
+    projectName = new JBCheckBox("Append VCS branch name to project name (Experimental)");
```

Comment:
![image](https://user-images.githubusercontent.com/25821770/143197122-241a8461-1cfb-4bb0-bb5a-5bb753235e93.png)

Reply:
Should this be the radio button component instead?
https://jetbrains.design/intellij/controls/radio_button/

Reply:
We change 
1. just window title
or
2. window title and project name

If only project name is changed - window title will be not contain branch name

Reply:
SGTM 👍